content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,255 @@ weight: 1
+{{< /text >}}
+
+Deploy a waypoint proxy for the `productpage` service:
+
+{{< text bash >}}
+$ istioctl x waypoint apply --service-account bookinfo-productpage
+{{< /text >}}
+
+View the `productpage` waypoint proxy status; you should see the details of the gateway
+resource with `Ready` status:
+
+{{< text bash >}}
+$ kubectl get gtw bookinfo-productpage -o yaml
+{{< /text >}}
+
+Verify that the waypoing proxy status is ready:
+
+{{< text plaintext >}}
+...
+status:
+  conditions:
+  - lastTransitionTime: "2023-02-24T03:22:43Z"
+    message: Deployed waypoint proxy to "default" namespace for "bookinfo-productpage"
+  	service account
+    observedGeneration: 1
+    reason: Ready
+    status: "True"
+    type: Ready
+{{< /text >}}
+
+Update our `AuthorizationPolicy` to explicitly allow the `sleep` service account and `istio-ingressgateway` service accounts to `GET` the `productpage` service, but perform no other operations:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: productpage-viewer
+ namespace: default
+spec:
+ selector:
+   matchLabels:
+     istio.io/gateway-name: bookinfo-productpage
+ action: ALLOW
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep", "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"]
+   to:
+   - operation:
+       methods: ["GET"]
+EOF
+{{< /text >}}
+
+Confirm the above authorization policy is working:
+
+{{< text bash >}}
+$ # this should fail with an RBAC error because it is not a GET operation
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ -X DELETE
+$ # this should fail with an RBAC error because the identity is not allowed
+$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/  | head -n1
+$ # this should continue to work
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | head -n1
+{{< /text >}}
+
+## Control Traffic {#control}
+Deploy a `waypoint` proxy for the review service, using the `bookinfo-review` service account, so that any traffic going to the review service will be mediated by the waypoint proxy.
+
+{{< text bash >}}
+$ istioctl x waypoint apply --service-account bookinfo-reviews
+{{< /text >}}
+
+Apply the reviews virtual service to control 90% traffic to reviews v1 and 10% traffic to reviews v2.
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/virtual-service-reviews-90-10.yaml
+$ kubectl apply -f samples/bookinfo/networking/destination-rule-reviews.yaml
+{{< /text >}}
+
+Confirm that roughly 10% traffic from the 100 requests go to reviews-v2:
+
+{{< text bash >}}
+$ kubectl exec -it deploy/sleep -- sh -c 'for i in $(seq 1 100); do curl -s http://istio-ingressgateway.istio-system/productpage | grep reviews-v.-; done'
+{{< /text >}}
+## Uninstall {#uninstall}
+Refer to our [uninstall instruction](/docs/setup/getting-started/#uninstall) to uninstall Istio and bookinfo sample application.
```

Comment:
Also need to uninstall sleep/nonsleep. Why not just put the uninstall commands here?

Reply:
and auth policy

Reply:
Updated!  :)